go/common/metrics/README.md

```diff
@@ -3,3 +3,34 @@ This package contains code related to the node's host metrics system.
 The metrics package heavily uses geths own metric package.
 - golang Package [here](https://github.com/ethereum/go-ethereum/tree/master/metrics)
 - Documentation [here](https://geth.ethereum.org/docs/interface/metrics)
+
+
+## Add metrics
+
+Instrumentation can be added and used as the following example shows:
+
+```go
+
+import gethmetrics "github.com/ethereum/go-ethereum/metrics"
+
+type SomeStruct struct {
+	...
+    someMeter gethmetrics.Meter
+	someGauge gethmetrics.Gauge
```

Comment:
formatting is dodgy

Reply:
Weird.. looked fine. Used only tabs now, seems to have fixed it.